src/components/PeoplePage.tsx

```diff
@@ -1,35 +1,174 @@
-import { PeopleFilters } from './PeopleFilters';
+import { useSearchParams } from 'react-router-dom';
+import { useEffect, useState } from 'react';
 import { Loader } from './Loader';
+import { Person } from '../types';
+import { getPeople } from '../api';
 import { PeopleTable } from './PeopleTable';
+import { PeopleFilters } from './PeopleFilters';
+
+function sortPeople(
+  people: Person[],
+  query: string,
+  sex: string,
+  centuries: string[],
+  sort: string,
+  order: string,
+) {
+  let updatedPeople = [...people];
+
+  switch (sort) {
+    case 'name':
+      updatedPeople = updatedPeople.sort((personA, personB) => {
+        return order === 'desc'
+          ? personB.name.localeCompare(personA.name)
+          : personA.name.localeCompare(personB.name);
+      });
+      break;
+
+    case 'sex':
+      updatedPeople = updatedPeople.sort((personA, personB) => {
+        return order === 'desc'
+          ? personB.sex.localeCompare(personA.sex)
+          : personA.sex.localeCompare(personB.sex);
+      });
+      break;
```

Comment:
you can combine some cases (for example: personB[sort].localeCompare(personA[sort]))